dashboard/app/views/home/_survey.html.haml

```diff
@@ -21,7 +24,7 @@
                 Of your students using Code.org's courses, how many are
           .dots{style: "padding-top: 10px; padding-left: 20px; padding-bottom: 20px"}
             - SurveyResult::ETHNICITIES.each do |key, value|
-              %input{type: "number",  style: "width:40px; font-size: 16px; text-align: right; margin: 5px; margin-left: 10px", name: "survey[survey2016_ethnicity_#{key}]", onclick: "enableSurveySubmit()", placeholder: "0"}
```

Comment:
The number-only thing felt clunky to use in Chrome so I ended up turning it off.